migration-engine/migration-engine-tests/tests/errors/error_tests.rs

```diff
@@ -22,7 +22,10 @@ async fn authentication_failure_must_return_a_known_error_on_postgres() {
         url
     );
 
-    let error = RpcApi::new(&dm).await.map(|_| ()).unwrap_err();
+    let error = RpcApi::new(&dm, GateKeeper::allow_all_whitelist())
+        .await
+        .map(|_| ())
```

Comment:
I don't remember what we have these `.map()` with a toilet closure for. Anyway, it's off-topic for this PR.